glide.lock

```diff
@@ -271,6 +271,7 @@ imports:
   - kubernetes/typed/storage/v1/fake
   - kubernetes/typed/storage/v1beta1
   - kubernetes/typed/storage/v1beta1/fake
+  - pkg/api/v1
```

Comment:
Hmmm actually @ianchakeres, the other external provisioners are logging PV events already, but they don't have to pull in all these vendoring changes.  They are hardcoding the event name, so I think that's why they didn't need to pull that in.  Can we do that here too?